.github/workflows/qrisp_test.yml

```diff
@@ -27,10 +26,12 @@ jobs:
           pip install -e ./
           pip install --upgrade pennylane-catalyst
           pip install qiskit-aer
-          pip install qiskit-iqm
+          pip install qiskit-ibm-runtime   # To test `QiskitBackend` with `qiskit_ibm_runtime.fake_provider`
```

Comment:
You can make these optional dependencies on qrisp specific for testing so that you don't need to do the install explicitly in the yml